docs/important/relevant_functionalities_de.md

```diff
@@ -1072,36 +1126,126 @@ Um eine Limitierung vornehmen zu können, muss die `dogu.json` des Dogus folgend
         "Type": "BINARY_MEASUREMENT"
       }
     },
+    {
+      "Name": "container_config/memory_request",
+      "Description":"Requests the container's minimal memory requirement. Use a positive integer value followed by one of these units [b,k,m,g] (byte, kibibyte, mebibyte, gibibyte).",
+      "Optional": true,
+      "Validation": {
+        "Type": "BINARY_MEASUREMENT"
+      }
+    },
     {
       "Name": "container_config/swap_limit",
       "Description":"Limits the container's swap memory usage. Use zero or a positive integer value followed by one of these units [b,k,m,g] (byte, kibibyte, mebibyte, gibibyte). 0 will disable swapping.",
       "Optional": true,
       "Validation": {
         "Type": "BINARY_MEASUREMENT"
       }
+    },
+    {
+      "Name": "container_config/cpu_core_limit",
+      "Description":"Limits the container's CPU core usage. Use a positive floating value describing a fraction of 1 virtual CPU core. When you define a value of '0.5', you are requesting half as much CPU time compared to if you asked for '1.0' CPU.",
```

Comment:
It can be virtual but it doesn't have to be.
```suggestion
      "Description":"Limits the container's CPU core usage. Use a positive floating value describing a fraction of 1 CPU core. When you define a value of '0.5', you are requesting half as much CPU time compared to if you asked for '1.0' CPU.",
```